packages/govuk-frontend-review/src/views/full-page-examples/make-a-payment/index.mjs

```diff
@@ -0,0 +1,37 @@
+import express from 'express'
+import { body, matchedData, validationResult } from 'express-validator'
+
+import { formatValidationErrors } from '../../../utils.mjs'
+
+const router = express.Router()
+
+router.post(
+  '/make-a-payment',
+
+  body('amount')
+    .notEmpty()
+    .withMessage('Enter an amount')
+    .isNumeric()
+    .withMessage('Enter a number')
+    .isFloat({ gt: 0 })
+    .withMessage('Enter an amount greater than 0'),
+
+  (req, res) => {
+    const { example } = res.locals
+
+    const viewPath = `./full-page-examples/${example.path}`
+    const errors = formatValidationErrors(validationResult(req))
+
+    if (!errors) {
+      return res.redirect(303, `./${example.path}/confirm`)
+    }
+
+    res.render(`${viewPath}/index`, {
+      errors,
+      errorSummary: Object.values(errors),
+      values: matchedData(req, { onlyValidData: false }) // In production this should sanitized.
```

Comment:
```suggestion
      values: matchedData(req, { onlyValidData: false }) // In production this should be sanitized.
```

Reply:
I'm going to be a meanie and reject this change, if just that this is a copy-pasted piece of code that has the same typo in about a dozen other places.